src/main/java/org/isf/security/jwt/TokenProvider.java

```diff
@@ -71,15 +73,21 @@ public class TokenProvider implements Serializable {
 
 	private JwtParser jwtParser;
 
+	@Autowired
+	private UserDetailsServiceImpl userDetailsService;
+
 	@PostConstruct
 	public void init() {
 		String secret = env.getProperty("jwt.token.secret");
 		LOGGER.info("Initializing JWT key with secret: {}", secret);
 		byte[] keyBytes = secret.getBytes(StandardCharsets.UTF_8);
 		this.key = Keys.hmacShaKeyFor(keyBytes);
 
-		this.tokenValidityInMilliseconds = 1000L * 6000;
-		this.tokenValidityInMillisecondsForRememberMe = 1000L * 6000;
+		// 15 minutes (900,000 milliseconds)
```

Comment:
15 minutes is too short I think. Maybe 30 minutes or 1 hour.

Reply:
Thanks! I will make them configurable anyway